### ISSUE TITLE
Focus trap

### ISSUE THREAD

**Issue by aloayzab88** [marekdedic/imagelightbox#1207]
In there any way to focus trap when the modal is open. Currently you can move outside the modal when navigating with the keyboard. The focus should be trapped inside the modal

**Comment by marekdedic** [comment]
Hi,
thanks for opening the issue. You mean when the image is open, pressing Tab should not focus elements in the rest of the page? Can you give me a more precise replication of the issue (steps taken, what you expect to happen and what actually happens)?

**Comment by aloayzab88** [comment]
Hi, yes. When using the keyboard to open the lightbox the focus should move to the first focusable element and trapped there.
Right now if you open a lightbox with enter the focus stays on the trigger element. Is you press tab the focus is moved to the next element outside the lightbox
